client/utils/account-fees.tsx

```diff
@@ -222,17 +222,17 @@ export const formatAccountFeesDescription = (
 		...customFormats,
 	};
 
+	let displayFeePercentageRate = baseFee.percentage_rate;
+	if ( displayFeePercentageRate <= 0 ) {
```

Comment:
Can we add a comment explaining this logic for clarity?

Reply:
Added.